docs/install.rst

```diff
@@ -43,132 +52,125 @@ To test that the configuration is working from the python browser, you can perfo
 
-We'll refer to that key as ``XXX`` in the following instructions.
 
-The simplest way to link your account is by typing 
+Package Version Management
+----------------------------
+
+Tidy3D and its dependencies can be installed from the command line via ``pip``, which is installed with Python when the new environment is created. Simply run
 
 .. code-block:: bash
 
-    tidy3d configure
+    pip install tidy3d
 
-and pasting the API key when prompted. Note that one can also specify the API key directly in the configure command as
+and the latest version of Tidy3D will be installed in this environment. To test whether the installation was successful you can run
 
 .. code-block:: bash
 
-    tidy3d configure --apikey=XXX
+    python -c "import tidy3d as td; print(td.__version__)"
 
-Note: Windows users will need to perform a slightly different step to link the registration. From the anaconda prompt where tidy3d was pip installed, the following commands should be run instead
+If the installation is successful, you should see the client version of Tidy3D being displayed. Now you can open your favorite Python IDE and start creating Tidy3D simulations!
+
+To get a specific version eg. ``x.y.z`` of tidy3d, including the "pre-release" versions, one may specify the version as follows:
 
 .. code-block:: bash
 
-    pip install pipx
-    pipx run tidy3d configure --apikey=XXX
+    pip install tidy3d==x.y.z
+
+The documentation for the most recent release is marked as "latest" and is available `here <https://docs.flexcompute.com/projects/tidy3d/en/latest/>`__. The documentation page also allows one to select the state of the docs based on version by toggling the dropdown in the bottom left corner.
 
-Alternatively, the API key can be set up using the environment variable ``SIMCLOUD_APIKEY`` as:
 
-.. code-block:: bash
+Advanced Installation Instructions
+==================================
 
-    export SIMCLOUD_APIKEY="XXX"
+Some users or systems may require a more specialized installation, which we will cover below.
 
-Finally, one may manually set the API key directly in the configuration file where Tidy3D looks for it.
+.. tabs::
 
-The API key must be in a file called ``.tidy3d/config`` located in your home directory, with the following contents
+    .. group-tab:: Conda/Mamba |:snake:|
 
-.. code-block:: bash
+        If you already have Python installed on your computer, it is possible that some packages in your current environment could have version conflicts with Tidy3D. To avoid this, we strongly recommend that you create a clean Python virtual environment to install Tidy3D.
 
-    apikey = "XXX"
+        We recommend using the Conda package management system to manage your Python virtual environment as well as installing Tidy3D. You can install Conda conveniently via `Anaconda <https://www.anaconda.com/>`__.
```

Comment:
hm, I guess we do say this in our current installation instructions. Although I've personally found conda a bit annoying to use, although I dont remember the specific details about it now. What are your thoughts on conda? maybe it's the best for windows users but not sure about for everyone else?

Reply:
We could update it to mamba which is better.  Conda on it's own is old and not great, and very slow. 

Maybe it's  a common enough tool that we should provide instructions! It's good for notebooks and people tinkering, but for reproducible development, poetry has been much better.